public/js/login.js

```diff
@@ -58,6 +56,40 @@ let loginUser = (event) => {
   })
 }
 
+//check to see if values in object are empty.
+let validateUser = (user) => {
+  if(user.email && user.user_password) {
+    return true;
+  }
+  return false;
+};
+
+let automaticSignIn = () => {
+  let checkUser;
+  if(localStorage.getItem("token")) {
+    checkUser = localStorage.getItem("token");
+    fetch(`/signedin`, {
```

Comment:
fetch request should be to '/api/signedin'.  I am getting a 404 error.

Reply:
i created a new route instead of /api

Reply:
Second this. authRouter only runs if the url starts with /api